### DISCUSSION TITLE
Compute Isotropic Elastic Tensor

### DISCUSSION THREAD

**Discussion by bbist196** [idaholab/moose#27356] [Q&A Modules: Solid mechanics]
Hello Team,

I am trying to work with temperature and concentration dependent isotropic elastic tensor. I was able to compute isotropic elastic tensor dependent on temperature using 'ComputeVariableIsotropicElasticityTensor' using temperature as 'args'. However, i want the isotropic elastic constants (tensor) which depends on both temperature and concentration. 

Do any one have any idea??

Help will be highly appreciated.

Thank you,
Bashant Bist

**Comment by GiudGiud** [comment]
Hello

Unless you can find another elasticity tensor material that does what you want, you will have to create a new material.

The variableIsotropic..Tensor is a good example to start from

Guillaume
